db/index.js

```diff
@@ -0,0 +1,50 @@
+//https://mongoosejs.com/docs/index.html
+const set = require('./complete.js');
+
+const mongoose = require('mongoose');
```

Comment:
No need for whitespace between imports, unless you're grouping them in a specific way. Also, let's keep them in alphabetical order.